keyboards/keebfront/vanguard65/config.h

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2022 <hello@keebfront.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *         ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+
```

Comment:
```suggestion
```

Reply:
Why is this being removed? @waffle87

Reply:
1. The include isn't needed (eg. #19714)
2. The matrix size is automatically generated, thus defining it manually isn't needed.
3. The comment provides little benefit to the end user, and is available elsewhere in the official documentation.

Reply:
Lovely, thank you for the information. Didn't realize they included the automatic matrix size generation and that PR.